UnitTests/MockDefaultValueProviderFixture.cs

```diff
@@ -93,6 +93,15 @@ public void CreatedMockIsVerifiedWithOwner()
 			Assert.Throws<MockVerificationException>(() => mock.Verify());
 		}
 
+		[Fact]
+		public void DefaultValueIsNotChangedWhenPerformingInternalInvocation()
+		{
+			var mockBar = new Mock<IBar> { DefaultValue = DefaultValue.Empty };
```

Comment:
I think that the MockDefaultValueProvuderFixture is better place for this test, as the EmptyDefalutValueProviderFixture tests the EmptyDefaultValueProvider class - unrelated to the scenario.